src/Data/Path/Pathy/Gen.purs

```diff
@@ -0,0 +1,58 @@
+module Data.Path.Pathy.Gen
+  ( genAbsDirPath
+  , genAbsFilePath
+  , genAbsAnyPath
+  , genRelDirPath
+  , genRelFilePath
+  , genRelAnyPath
+  )where
+
+import Prelude
+
+import Control.Monad.Gen (class MonadGen)
+import Control.Monad.Gen as Gen
+import Control.Monad.Rec.Class (class MonadRec)
+import Data.Char.Gen as CG
+import Data.Either (Either(..))
+import Data.Foldable (foldr)
+import Data.List as L
+import Data.NonEmpty ((:|))
+import Data.Path.Pathy (AbsPath, AbsFile, AbsDir, RelDir, RelFile, RelPath, Sandboxed, (</>))
+import Data.Path.Pathy as P
+import Data.String.Gen as SG
+
+genName ∷ ∀ m. MonadGen m ⇒ MonadRec m ⇒ m String
+genName = SG.genString $ Gen.oneOf $ CG.genDigitChar :| [CG.genAlpha]
```

Comment:
https://github.com/purescript-contrib/purescript-strongcheck/blob/master/src/Test/StrongCheck/Data/AlphaNumString.purs#L10

Reply:
But we have no dependency on that.